.github/CODEOWNERS

```diff
@@ -82,11 +80,24 @@ pyproject.toml                      @jnm @noliveleger
 
 
 ################
-#### Infra section
+#### Infrastructure section
 
 docker/*                            @bufke @jnm @noliveleger
 scripts/*                           @bufke @jnm @noliveleger
 .github/*                           @bufke @jnm @noliveleger @magicznyleszek
 .gitlab-ci.yml                      @bufke
 Dockerfile                          @bufke @jnm @noliveleger
 
+
+
+
+################
+#### Documentation section
+CONTRIBUTING.md                     @Akuukis @magicznyleszek @noliveleger
+README.md                           @Akuukis @jnm @magicznyleszek @noliveleger
+.github/CODEOWNERS.md               @Akuukis @jnm @noliveleger @magicznyleszek
+.github/FUNDING.md                  @Akuukis @jnm @noliveleger @magicznyleszek
+.github/ISSUE_TEMPLATE.md           @Akuukis @jnm @noliveleger @magicznyleszek
+.github/PULL_REQUEST_TEMPLATE.md    @Akuukis @jnm @noliveleger @magicznyleszek
```

Comment:
Should we add these too?
- jsapp/js/hooks/README.md
- jsapp/scss/libs/README.md
- jsapp/js/components/processing/README.md

Reply:
good question, maybe, let's keep it out of scope - for now I did the minimal detour `.github/*` to start owning this file.